core/block_svg.ts

```diff
@@ -1184,6 +1184,10 @@ export class BlockSvg
     /* eslint-disable-next-line @typescript-eslint/no-this-alias */
     let block: this | null = this;
     do {
+      const isDeadOrDying: boolean = block.isDeadOrDying();
+      if (isDeadOrDying) {
```

Comment:
If you're immediately checking it, there's no need to store the value in a temp variable!
```suggestion
      if (block.isDeadOrDying()) {
```